test_scripts/Resumption/Handling_errors_from_HMI/025_resume_all_data_with_one_error_rpc_unexpected_disconnect_background.lua

```diff
@@ -0,0 +1,69 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0190-resumption-data-error-handling.md
+--
+-- Description:
+-- In case:
+-- 1. App is in BACKGROUND HMI level
+-- 2. AddCommand, AddSubMenu, CreateInteractionChoiceSet, SetGlobalProperties, SubscribeButton, SubscribeVehicleData,
+--  SubscribeWayPoints, CreateWindow (<Rpc_n>) are added by app
+-- 3. Unexpected disconnect and reconnect are performed
+-- 4. App re-registers with actual HashId
+-- SDL does:
+--  - start resumption process
+--  - send set of <Rpc_n> requests to HMI
+-- 5. HMI responds with <erroneous> resultCode to one request and <successful> for others
+-- SDL does:
+--  - process responses from HMI
+--  - remove already restored data
+--  - send set of revert <Rpc_n> requests to HMI (except the one related to <erroneous> response)
+--  - respond RegisterAppInterfaceResponse(success=true,result_code=RESUME_FAILED) to mobile application
+--  - not restore BACKGROUND HMI level
```

Comment:
Check do we need restore BACKGROUND is resumption is success

Reply:
In case if resumption has finished with success result code SDL does not restore `BACKGROUND` HMI level.
Hence data resumption is not related to HMI level resumption.